examples/scenes/humanoid.xml

```diff
@@ -65,9 +67,9 @@
       </default>
 
       <!-- joints -->
-      <joint type="hinge" damping=".4" stiffness="2" armature=".01" limited="true" solimplimit="0 .99 .01"/>
+      <joint type="hinge" damping=".2" stiffness="1" armature=".01" limited="true" solimplimit="0 .99 .01"/>
       <default class="joint_big">
-        <joint damping="10" stiffness="20"/>
+        <joint damping="5" stiffness="10"/>
```

Comment:
Did you retrain the original policy with these values?

Reply:
nah

Reply:
Does the policy work then without changing these parameters?

Reply:
yea

Reply:
So why changing them?

Reply:
uh forgot to change back woops